pkg/dhclient/dhclient.go

```diff
@@ -27,6 +28,32 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// DisableSLAAC disable IPv6 auto-configure on each interface.
+//
+// As soon as the interface is set up, the kernel (by default) will try to
+// auto-configure global addresses on it using SLAAC. This settings does not
+// affect link-local SLAAC addresses.
+//
+// SLAAC would mean a global address will magically appear on the interface a
+// few seconds later which is different from the address returned by DHCPv6.
+func DisableSLAAC(ifs ...netlink.Link) error {
+	for _, iface := range ifs {
+		name := iface.Attrs().Name
+		autoconfPath := filepath.Join("/proc/sys/net/ipv6/conf", name, "autoconf")
+		if err := ioutil.WriteFile(autoconfPath, []byte("0\n"), 0777); err != nil {
+			return fmt.Errorf("error disabling autoconf on %s: %v", name, err)
+		}
+
+		// Make sure Router Advertisement (RA) remains enabled. The routes are being
```

Comment:
I know nothing about how this works but ... even if the autoconf fails, is there a chance you want to try the accept_ra command?

Reply:
accept_ra should be unconditionally set to 1. It's the only way to get routes in ipv6, and we don't have a userspace implementation in u-root.